processing/src/main/java/io/druid/query/aggregation/AggregatorFactory.java

```diff
@@ -63,6 +59,20 @@
   public abstract Object combine(Object lhs, Object rhs);
 
   /**
+   * Creates an AggregateCombiner to combine rollup aggregation results from serveral "rows" of different indexes during
+   * index merging. AggregateCombiner implements the same logic as {@link #combine}, with the difference that it uses
+   * {@link io.druid.segment.ColumnValueSelector} and it's subinterfaces to get inputs and implements {@code
+   * ColumnValueSelector} to provide output.
+   *
+   * @see AggregateCombiner
+   * @see io.druid.segment.IndexMerger
+   */
+  public AggregateCombiner makeAggregateCombiner()
+  {
+    throw new UOE("[%s] does not implement makeAggregateCombiner()", this.getClass().getName());
```

Comment:
It's not possible to provide a safe default implementation, because delegating to `combine()` may modify source objects, that keeps the #4672 bug alive and it's why AggregateCombiner prohibits mutation. OTOH we don't know how to clone the input object for any kind of object.